### DISCUSSION TITLE
关于Modrinth的问题

### DISCUSSION THREAD

**Discussion by zhoujy1209** [Meloong-Git/PCL#1149] [提问] [answered]
什么时候弄好哇~

**Comment by WForst-Breeze** [comment, marked as answer]
请耐心等待，目前 Modrinth 资源下载已经由 **暂缓处理** 变更为 **处理中**，相信很快就会来的😍
![image](https://user-images.githubusercontent.com/110760354/216754427-964b2d81-19df-4281-82b0-8ae0d42ba8f7.png)
补充：请见[b站更新日志](https://www.bilibili.com/read/cv21661346)，下个版本就会添加对 Modrinth 的支持
